app/src/main/java/org/schabi/newpipe/local/feed/FeedFragment.java

```diff
@@ -71,6 +71,10 @@ public void onCreate(Bundle savedInstanceState) {
 
     @Override
     public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, Bundle savedInstanceState) {
+
+        if(!useAsFrontPage) {
```

Comment:
Same as with mIsVisibleToUser. This should maybe be put into `BaseFrament` then onCreateView() should call its super.